feat(core)!: Rename package to assertive-ts/core (#105)
stackbuilders/assertive-ts: README.md

- Works with any test runner and framework such as [Jest](/docs/jest-tutorial.md), [Mocha](/docs/mocha-tutorial.md), or Ava
- Well tested: more than 300 tests!

## Packages

> For conviniece, this library is split into package grouped within the same namespace:

Reviewer: ```suggestion
For convenience, this library is split into packages grouped within the same namespace:
```

> - **[assertive-ts/core](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md):** Core functionalities, assertions applicable for any kind of application. This package is required to use the [extension mechanism](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md#extension-mechanism-⚙️) (plugins). This is package is the replacement for the deprecated `stackbuilders/assertive-ts` package.

Reviewer: ```suggestion
- **[assertive-ts/core](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md):** Core functionalities, assertions applicable for any kind of application. This package is required for [extension mechanism](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md#extension-mechanism-⚙️) (plugins). Also, this package replaces the deprecated `stackbuilders/assertive-ts` package.
```

Reviewer: This suggestion here may not be worth as the same paragraph is duplicated in `core/readme`, and accepting this change my imply keeping the other one in sync with this one, so, up to you.

## Usage

> Using you favorite test runner, you just need to import the `expect` and test away! If you don't really agree with `expect` as the name of the assertion function, we provide a couple aliases to the function, such as `assert` and `assertThat`.

Reviewer: ```suggestion
Using you favorite test runner, you just need to import the `expect` and test away! If you don't really agree with `expect` as the name of the assertion function, we provide a couple aliases such as `assert` and `assertThat`.
```

Reviewer: This suggestion here may not be worth as the same paragraph is duplicated in `core/readme`, and accepting this change my imply keeping the other one in sync with this one, so, up to you.

Author: Makes sense, we have this part duplicate in the two readme files to provide better visibility of how the library works in GitHub as well as NPM 🙂
